paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/ReplayHandler.java

```diff
@@ -40,6 +44,13 @@ public void error(final String consumerName, final Throwable t, final EventTypeP
         }
     };
 
+    Predicate<EventTypeConsumer> filterConsumer(@Nonnull final String eventName, @Nullable final String consumerName) {
```

Comment:
As written before, you moved this method from `ControllerConfiguration` but don't use it here.